docs/lending/contracts/secondary-mp.md

```diff
@@ -0,0 +1,569 @@
+<h1>Secondary Monetary Policy</h1>
+
+The `SecondaryMonetaryPolicy` contract calculates borrow rates based on the utilization in a lending market. It uses parameters derived from the target utilization and ratios at 0% and 100% utilization to define a hyperbolic dependency. The rate is dynamically adjusted based on the current utilization and the rate from the AMM (Automated Market Maker), which mints crvUSD.
+
+This design ensures that when the target utilization is met, the borrow rate in the lending market matches the borrow rate of the minting market. At 0% utilization, the rate is defined as \(\alpha \times \text{rate}_{\text{AMM}}\) and at 100% utilization as \(\beta \times \text{rate}_{\text{AMM}}\).
+
+!!!github "GitHub"
+    The source code of the `SecondaryMonetaryPolicy.vy` contract can be found on [:material-github: GitHub](https://github.com/curvefi/curve-stablecoin/blob/master/contracts/mpolicies/SecondaryMonetaryPolicy.vy).
+
+---
+
+## **Calculations**
+
+!!!colab "Google Colab Notebook"
+    An interactive Google Colab notebook that plots the interest rate depending on utilization can be found here: [https://colab.research.google.com/drive/1lU0SWtvQoJHNe7pLiKD33nYBKacljhck?usp=sharing](https://colab.research.google.com/drive/1lU0SWtvQoJHNe7pLiKD33nYBKacljhck?usp=sharing).
+
+
+### **Borrow Rate**
+
+The formula for calculating the borrow rate is as follows:
+
+$$\text{rate} = \text{rate}_{\text{AMM}} \left( r_{\text{minf}} + \frac{A}{u_{\text{inf}} - \text{utilization}} \right) + \text{shift}$$
+
+$\text{shift}$ is an additional value which shifts the entire rate curve up or down by a specified amount.[^1]
+
+[^1]: This kind of rate shift is rarely used but is applied, for example, in the wstETH lending market. The `SecondaryMonetaryPolicy` of that market does not follow the wstETH mint market but follows the wETH mint market instead, with a +4% shift applied to the rate. This is done because the "more fair" interest rate is the wETH rate plus the staking rate (which is approximately 4%).
```

Comment:
so strange that wstETH lending doesn't follow wstETH minting

Reply:
yeah, not very intuitive but makes sense if you think about it. its more fair to follow the wETH mint market + add a shift of 4% (which is approx the staking rate), instead of directly following the wstETH mint market.

here is mich's explainer: https://t.me/crvtraders/351232